core/modules/user/user.pages.inc

```diff
@@ -559,17 +559,17 @@ function user_page() {
   global $user;
 
   // To help users upgrading from versions of Backdrop prior to 1.9.0 where this
-  // menu callback was used on 'user/login' as well as 'user', rebuild the
-  // menu automatically if this page is loaded at 'user/login'.
+  // menu callback was used on 'login' as well as 'user', rebuild the
+  // menu automatically if this page is loaded at 'login'.
   // See https://github.com/backdrop/backdrop-issues/issues/260.
-  if ($_GET['q'] === 'user/login') {
+  if ($_GET['q'] === 'login') {
```

Comment:
This is here for backwards compatibility. I don't think that it should be changed.